facebookads/adobjects/adreportrun.py

```diff
@@ -113,14 +116,8 @@ def get_insights(self, fields=None, params=None, async=False, batch=None, pendin
         if async:
           return self.get_insights_async(fields, params, batch, pending)
         param_types = {
-            'default_summary': 'bool',
-            'fields': 'list<fields_enum>',
-            'filtering': 'list<Object>',
-            'sort': 'list<string>',
-            'summary': 'list<summary_enum>',
```

Comment:
@m-vdb 
so here is the main changing thing I've seen in the changelog and it's consistent to the [changelog announce](https://developers.facebook.com/docs/apps/changelog). I thought this would change totally the way we query the insights API, but I just made a test with the [ad_reporting_query](https://github.com/Work4Labs/work4us/blob/4447399d02f6756c218d751dfcc54e615244c24f/python/ads_app/parameters.py#L107-L144) and it seems like it worked the same... So I'm not sure if I'm missing something here or not.